obspy/io/reftek/core.py

```diff
@@ -241,10 +241,14 @@ def to_stream(self, network="", location="", component_codes=None,
             else:
                 eh = EHPacket(et_packets[0])
             # only "C0" encoding supported right now
-            if eh.data_format != b"C0":
+            if eh.data_format == b"C0":
+                encoding = 'C0'
+            elif eh.data_format == b"C2":
+                encoding = 'C2'
```

Comment:
I think it's OK if it stays as is. Too minor to worry about. Mainly I'm not doing decoding because then I would have to worry about decoding errors (ignore? replace? mainly for potentially garbled files..). Right now the user would get warned about the unrecognized encoding as a bytestring exactly as it is found in the file..

But like I said, too minor to give it more thought, really..